src/Evolution/Systems/Cce/Tags.hpp

```diff
@@ -0,0 +1,255 @@
+// Distributed under the MIT License.  See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/SpinWeighted.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace Cce {
+
+/// Tags for Cauchy Characteristic Extraction routines
+namespace Tags {
+
+// Bondi parameter tags
+
+/// Bondi parameter \f$\beta\f$
+struct Beta : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, 0>>;
+  static std::string name() noexcept { return "Beta"; }
+};
+
+/// Bondi parameter \f$J\f$
+struct H : db::SimpleTag {
```

Comment:
I have a mild preference for keeping the tags brief, due to the effect of expanding the already very long tag lists in `Equations.hpp`. The names are consistently used in CCE, and anything that accesses these tags outside the CCE module will need to access them by `Cce::Tags::H` etc, so I think it is sufficiently disambiguated.

Reply:
I’d still prefer `BondiH`, `BondiJ` etc, just to avoid single-letter type names that have the possibility of being hard to search-and-replace. I’d also like the other reviewers @markscheel @nilsdeppe to weigh in here before you make the change.